config.py

```diff
@@ -13,3 +17,16 @@ class Config(object):
     SESSION_COOKIE_HTTPONLY = True
```

Comment:
Are we sure this won't mess up local development by enforcing HTTPS locally?

Reply:
local dev was working with it but is also without and I can't see where it's being used and it isn't present in pre-award-frontend so I'll just remove for now... there were a bunch of env vars i copied across from somewhere when I was first trying to get it set up...